### ISSUE TITLE
Add stickers 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#396]
frog 
hazard stickers
something like a radioactive sticker, or "this is potentially very laggy" sticker that you put as deco

OgelGames 
I should probably finish these at some point... (I can't even remember what was wrong with them)

frog 
make a dumpster fire sticker too

SX 
One thing I find wrong with them: not blocky/minetesty enough. In my opinion needs some work, at minimum cut resolution to some fraction but probably could do some other styling too.
Maybe yellow warning signs are almost there or even right there for apparent resolution 🤔  Other signs especially ones with small detail are way too... well, detailed.

**Comment by TangentFoxy** [comment]
Is there a mod that allows one to display an image like a painting/sign using a URL? I've seen a similar thing for Minecraft but I don't know if it exists here. Such a mod would be a good way to implement this without restricting what things people could display.

**Comment by SwissalpS** [comment]
Using a NIC (digistuff:nic) one could fetch small images from the web. Using digistuff:gpu and the display from was it cheapie, one could essentially have url-based signs.

But I don't see that for Pandorabox. Also I don't think those higher resolution items fit well into the game concept.

**Comment by TangentFoxy** [comment]
I agree. Something that could do that but with a resolution limit would be nice.

I forgot what the POI block URL option does, but is that something that could used as a starting point?

**Comment by SwissalpS** [comment]
All the fields of the POI only have a function to mapserver or players that open the formspec in-game.
